libsrc/c64/acc_detect_turbomaster.s

```diff
@@ -0,0 +1,45 @@
+;
+; Marco van den Heuvel, 2018-04-30
+;
+
+; unsigned char detect_turbomaster (void);
+;
+;/* Check for the presence of a Turbo Master cartridge.
+; *
+; * Possible return values:
+; * 0x00  : TurboMaster cartridge not present
+; * 0x01  : TurboMaster cartridge present
+; */
+
+        .export         _detect_turbomaster
+
+        .include        "accelerator.inc"
+
+_detect_turbomaster:
+        lda     #$00
+        tax
+
+; Make sure the current CPU is not a 6510
+        .byte   $1A                   ; NOP on 8502, INA on 65(S)C(E)02, 4510 and 65816
```

Comment:
No prerequisite for a merge but maybe you want to harmonize the the comment-start-column. It differs here from the ones below.